web_programming/instagram_pic.py

```diff
@@ -36,7 +36,7 @@ def download_image(url: str) -> str:
     if not image_data:
         return f"Failed to download the image from {image_url}."
 
-    file_name = f"{datetime.now():%Y-%m-%d_%H:%M:%S}.jpg"
+    file_name = f"{datetime.now(tz=UTC):%Y-%m-%d_%H:%M:%S}.jpg"
```

Comment:
I would think the user would expect us to use their local timezone here.
```suggestion
    file_name = f"{datetime.now(tz=UTC).astimezone():%Y-%m-%d_%H:%M:%S}.jpg"
```